scriptmodules/libretrocores/lr-mame2003-plus.sh

```diff
@@ -12,21 +12,61 @@
 rp_module_id="lr-mame2003-plus"
 rp_module_desc="Arcade emu - updated MAME 0.78 port for libretro with added game support"
 rp_module_help="ROM Extension: .zip\n\nCopy your MAME roms to either $romdir/mame-libretro or\n$romdir/arcade"
-rp_module_licence="NONCOM https://raw.githubusercontent.com/libretro-mirrors/mame2003-plus-libretro/master/docs/mame.txt"
-rp_module_section="exp"
+rp_module_licence="NONCOM https://github.com/arcadez/-mame2003-plus-libretro/blob/master/docs/mame.txt"
+rp_module_section="main"
 
 function sources_lr-mame2003-plus() {
-    gitPullOrClone "$md_build" https://github.com/libretro-mirrors/mame2003-plus-libretro.git
+    gitPullOrClone "$md_build" https://github.com/arcadez/-mame2003-plus-libretro
```

Comment:
why the hyphen repo prefix in the repo name btw?

Reply:
When i forked the plus mirror the git did the naming

Reply:
That's strange. Never seen that before when forking.

Are you planning on keeping the libretro github up to date or is your repo meant to replace it ?

Reply:
Hmm come to think of it i might have renamed it after forking it was a while ago and well
,my memory ain't what it was :)

With regards to your question i have maintained both cores simultaneously they lineup codewise 
the only difference being just incase i made a mistake with the folder renaming i've not added those changes to the mame2003-plus mirror as yet.

Anyway look as i said with regards to the retropie setup files feel free to hook up with whatever
core you would prefer it doesn't have to be mine just lemmie know if you choose the mirror
and i'll update it for that purpose.

Regards!!